Tensile/SolutionLibrary.py

```diff
@@ -500,33 +509,38 @@ def applyNaming(self, naming=None):
         for s in list(self.solutions.values()):
             s.name = OriginalSolution.getNameMin(s.originalSolution.getKernels()[0], naming)
 
-    def remapSolutionIndicesStartingFrom(self, curIndex):
+    def remapSolutionIndicesStartingFromImpl(self, library, solutions: dict, curIndex: int) -> tuple:
```

Comment:
```suggestion
    def _remapSolutionIndicesStartingFromImpl(self, library, solutions: dict, curIndex: int) -> tuple:
```
Add underscore to identify it as a private method?